src/analyses/callstringTracking.ml

```diff
@@ -0,0 +1,36 @@
+open Analyses
+
+module Spec : MCPSpec =
+struct
+  include Analyses.IdentitySpec
+
+  let name () = "callstringTracking"
```

Comment:
Why `callstringTracking`? Maybe `loopfreeCallstring` or something like this is better?